go/merklestore/store.go

```diff
@@ -268,23 +260,6 @@ func (s *MerkleStoreImpl) fetch(m libkb.MetaContext, hash keybase1.MerkleStoreKi
 	return res.KitJSON, nil
 }
 
-// updateRoot kicks MerkleClient to update its merkle root
```

Comment:
why was this code deleted?

Reply:
It is really old code which depends on t_alice being there to update the merkle_client's root. 
The same functionality is now provided by FetchRootFromServer (see other edits in the same file for the only place where it's used), so deleting duplicated code and simplifying seemed appropriate.